src/spectrecoff-cli/commands/Json.fs

```diff
@@ -0,0 +1,94 @@
+namespace SpectreCoff.Cli.Commands
+
+open Spectre.Console
+open Spectre.Console.Cli
+open SpectreCoff
+
+type JsonSettings()  =
+    inherit CommandSettings()
+
+type JsonExample() =
+    inherit Command<JsonSettings>()
+    interface ICommandLimiter<JsonSettings>
+
+    override _.Execute(_context, _settings) =
+
+        Many [
+            CO [ 
+                Emoji ":backhand_index_pointing_right:"
+                P "This example is taken directly from Spectre.Console" 
+                Emoji "astonished_face"
```

Comment:
I don't remember exactly how this `Emoji` union case (right term? :)) came into being, but why not just something like:

```
                E Emoji.Known.BackhandIndexPointingRight
                P "This example is taken directly from Spectre.Console"
                E Emoji.Known.AstonishedFace
```

It seems strange to me to use some more or less magic strings if we have them strongly typed as constants.

Reply:
The reason is simple - there is a bug using the constants. Look at this example: If you use the string literal you can use the `:`, or leave them, works both. If you use the constant, the emoji is padded with `:` on both sides :/
<img width="382" alt="image" src="https://user-images.githubusercontent.com/10219667/219469072-0ab9752b-d4c4-4aa7-ba2e-bd0b525fd9a9.png">

Reply:
Aha! That is interesting, and explains why it looked kind of strange when I implemented the grid:

![image](https://user-images.githubusercontent.com/17160319/219855485-08edbbcf-933f-48c7-97cc-102b9568eda2.png)

Anyway, thanks for explaining